client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx

```diff
@@ -189,6 +189,9 @@ export const ChapterUsersPage: NextPageWithLayout = () => {
                       }
                     >
                       Change
+                      <Text srOnly as="span">
+                        {otherUser.name} role
```

Comment:
```suggestion
                        role of {otherUser.name} 
```
Might be a touch clearer.  I considered `{otherUser.name}'s role`, but I suspect apostrophes are not the best when using text to speech.